test/support/pages/login_page.ex

```diff
@@ -0,0 +1,36 @@
+defmodule LoginPage do
+  use Hound.Helpers
+
+  def open do
+    navigate_to("#{Application.get_env(:hound, :torus_base_url)}/")
+  end
+
+  def close_cookie_prompt do
+    # Make sure the dialog & button are there
+    # Then click the accept button
+    with {:ok, _} <- search_element(:xpath, "//*[text() = 'We use cookies']"),
+         {:ok, button} <- search_element(:xpath, "//*[text() = 'Accept']") do
+      click(button)
+    end
+  end
+
+  def go_to_author_login do
+    click({:link_text, "Authoring Sign In"})
+  end
+
+  def go_to_educator_login do
+    # TODO - It would be better for these to have explicit id or aria-label attributes instead of relying on the implemention-detail of the url they go to
+    click({:xpath, "//*[@href='/session/new']"})
```

Comment:
You can also use phoenix paths. eg: `Routes.pow_session_path(OliWeb.Endpoint, :new)`

Reply:
Updated to use Routes, thanks!